publish a pose relative to the first pose

chhtz opened the pull request:
As discussed today. I tested this and it works as intended, but I'm open to API changes.

Also, I did not add a `reset()` operation yet (this would only need to set `start_pose_initialized = false`).
It would also be possible to give the `reset` operation an optional parameter, to set a manual start pose.

chhtz committed:
publish a pose relative to the first pose

planthaber commented:
I think hitting reconfigure in rock-display as Sebastian suggested is more convenient compared with an operation. But for this to work, start_pose_initialized = false should be set in configureHook()

chhtz committed:
Reset start pose in configureHook.

chhtz committed:
Update vicon.orogen

Dropped the idea of providing a special operation to reset the relative pose output.

chhtz commented:
I changed the documentation for `relative_pose` (dropping the idea of a `reset_pose()` operation). 
From my point this can get merged as-is. @skasperski @planthaber
